lib/actual_db_schema.rb

```diff
@@ -26,9 +27,12 @@ class << self
   self.failed = []
   self.config = {
     enabled: Rails.env.development?,
-    auto_rollback_disabled: ENV["ACTUAL_DB_SCHEMA_AUTO_ROLLBACK_DISABLED"].present?
+    auto_rollback_disabled: ENV["ACTUAL_DB_SCHEMA_AUTO_ROLLBACK_DISABLED"].present?,
+    ui_disabled: Rails.env.production? || ENV["ACTUAL_DB_SCHEMA_UI_DISABLED"].present?
```

Comment:
```suggestion
    ui_enabled: Rails.env.development? || ENV["ACTUAL_DB_SCHEMA_UI_DISABLED"].present?
```

Let's revert this, I guess this thing can even be helpful for staging or other env. But we definitely know, that it's useful in one env and it's `development`. Only this should be the default one. Optionally, it can be enabled everywhere and env var is good for that. Moreover, `ui_enabled` corresponds with the convention above with `enabled`.